.github/workflows/stale-prs.yml

```diff
@@ -51,8 +51,13 @@ jobs:
                     per_page: 100,
                   });
                   
+                  if (pr.draft) {
```

Comment:
Should it cover undefined chance here or are we good to accept `pr` and `pr.draft` are not undefined at this point?

Reply:
github docs state `Draft pull requests are available in public repositories with GitHub Free and GitHub Free for organizations, GitHub Pro, and legacy per-repository billing` https://docs.github.com/en/rest/pulls/pulls?apiVersion=2022-11-28#list-pull-requests

So pr.draft should be not undefined at this point. and for pr also at this point it should not be undefined because it would have failed at line 39 itself. we can add a check to verify pr is not undefined at an earlier point tho.